### PR TITLE
Include Mabry Pro font and fix fonts across tables

### PR DESCRIPTION
**Pull request by allysonjp715**
Screen Shot 2020-09-11 at 4 53 22 PM

### CODE REVIEW

File: verification/curator-service/ui/src/components/App.tsx [hunk 1]
```diff
@@ -95,6 +95,19 @@ const theme = createMuiTheme({
...
             },
         },
+        MuiTablePagination: {
+            root: {
+                border: 'unset',
+                fontFamily: 'Inter',
+                '& .MuiTablePagination-input': {
+                    fontFamily: 'Inter',
+                },
+                '&&& .MuiTypography-root': {
```
Comment (attwad) [inline verification/curator-service/ui/src/components/App.tsx:105]: sure it's not &&&&&& or &&&&&&&&& ? ;-)

File: verification/curator-service/ui/src/components/SourceTable.tsx [hunk 2]
```diff
@@ -535,21 +534,18 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                             Pagination: (props): JSX.Element => {
                                 return (
                                     <div className={classes.tablePaginationBar}>
-                                        <Typography>
+                                        <Typography style={{ width: '100%' }}>
```
Comment (attwad) [inline verification/curator-service/ui/src/components/SourceTable.tsx:537]: hmm shouldn't this be a className instead?

**Review by attwad — APPROVED** [review]
that users table though 😭

File: verification/curator-service/ui/src/components/SourceTable.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/SourceTable.tsx]: I figured it was small enough to use styles, but changed to classes for consistency.

**Comment by attwad** [comment]
> that users table though 😭

ah no it is included, I thought it was missing, great! 👍

**Pull request merged**